src/display/display_utils.js

```diff
@@ -407,19 +407,19 @@ class StatTimer {
   }
 
   toString() {
-    let times = this.times;
+    const times = this.times;
```

Comment:
Completely optional nit: With the `for...of` loops below I'm not sure if this aliasing is necessary any more, also considering that the method isn't even called by default, so you could probably just remove this and simply format the two loops below as `for (const time of this.times) {` instead.